javatests/arcs/core/host/AbstractArcHostTest.kt

```diff
@@ -28,8 +29,8 @@ import org.junit.runners.JUnit4
 open class AbstractArcHostTest {
 
     class TestParticle : BaseParticle() {
-        override val handles: HandleHolder =
-            HandleHolderBase("TestParticle", mapOf("foo" to EntityBaseSpec(DummyEntity.SCHEMA)))
+        override val handles: HandleHolder = HandleHolderBase(
+            "TestParticle", mapOf("foo" to setOf(EntityBaseSpec(DummyEntity.SCHEMA))))
```

Comment:
One arg per line, and close parens on new line:

```
HandleHolderBase(
    "TestParticle",
    mapOf("foo" to setOf(EntityBaseSpec(DummyEntity.SCHEMA)))
)
```

Reply:
Done, thanks!